docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,544 @@
+
+osnadmin channel join –-channel-id [CHANNEL_NAME]  --config-block [CHANNEL_CONFIG_BLOCK] -o [ORDERER_ADMIN_LISTENADDRESS] --ca-file $TLS_CA_ROOT_CERT --client-cert $OSN_TLS_SIGN_CERT --client-key $OSN_TLS_PRIVATE_KEY
+```
+
+Replace:
+- `CHANNEL_NAME` with the name you want to call this channel.
+- `CHANNEL_CONFIG_BLOCK` with the path and file name of the genesis block you created in Step one if you are creating the channel. Each subsequent ordering node can join the configuration starting with the genesis block, or they can join by providing the latest config block instead.
+- `ORDERER_ADMIN_LISTENADDRESS` corresponds to the `Orderer.Admin.ListenAddress` defined in the `orderer.yaml` for this orderer.
+- `TLS_CA_ROOT_CERT` with the path and file name of the orderer organization TLS CA root certificate and intermediate certificate if using an intermediate TLS CA.
+- `OSN_TLS_SIGN_CERT` with the path and file name of the orderer signed certificate from the TLS CA.
+- `OSN_TLS_PRIVATE_KEY` with the path and file name of the orderer private key from the TLS CA.
+
+For example:
+```
+osnadmin channel join –-channel-id channel1 --config-block genesis_block.pb -o OSN1.example.com:7050 --ca-file $TLS_CA_ROOT_CERT --client-cert $OSN_TLS_SIGN_CERT --client-key $OSN_TLS_PRIVATE_KEY
+```
+
+**Note:** The connection between the `osnadmin` CLI and the orderer requires mutual TLS, although it is not required for the network itself. This means you need to pass the `--client-cert` and `--client-key` parameters on each `osadmin` command. The `--client-cert` parameter points to the orderer certificate issued by the TLS CA, and `--client-key` refers to the orderer private key issued by the TLS CA.
+
+The output of this command looks similar to:
+```
+{
+  "name": "channel1",
+  "url": "participation/v1/channels/channel1",
+  "consensusRelation": "consenter",
+  "status": "active",
+  "height": 1
+}
+```
+
+When successful, the channel is started with a ledger height `1`. Because this orderer is joining from the genesis block, the `status` is "active". And because the orderer is part of the channel's consenter set, its `consensusRelation` is "consenter". We delve more into the `consensusRelation` value in the next section.
+
+You can repeat the same command on the other two orderers. Remember to update the `OSN_TLS_SIGN_CERT` and `OSN_TLS_PRIVATE_KEY` environment variables to point to the certificates for the corresponding orderer. Because these two orderers join the channel from the genesis block and are part of the consenter set, their status transitions almost immediately from  "onboarding" to "active" and the "consensusRelation" status is "consenter". At this point, the channel becomes operational because a majority of consenters (as defined in the channel configuration) are "active". You should see something similar to the following in your orderer logs:
```

Comment:
Again, these (the `OSN_TLS_SIGN_CERT` and `OSN_TLS_PRIVATE_KEY`) are actually the client's certificate and key. They can stay the same.

If all 3 orderers come from the same org and have the same ROOT CA, then the only thing that needs to change it the orderer endpoint: `-o OSN1.example.com:7050` -> `-o OSN2.example.com:7050`